tests/functions.ts

```diff
@@ -35,13 +37,16 @@ export const rndStr = () => Math.random().toString(36).substr(2);
 export const backupSettings = () => {
   // fs.copyFileSync isn't supported in Node 6/7
```

Comment:
We don't support Node 6/7 anymore since it's unsupported by Node itself. Please remove.